db/payment_icons.yml

```diff
@@ -1434,6 +1434,10 @@
   name: cebuana
   label: Cebuana
   group: other
+-
+  name: swissbilling
+  label: swissbilling
```

Comment:
The label should reflect the name of the brand, googling the brand it seems like it should be `SwissBilling` but can you validate the right brand naming guideline is used

Reply:
@rahilvirani This is intentional, we need to use "swissbilling". You can see we use small cases on our logo as well. Please approve it as it is.